recipes-extended/xdial/xdial.bb

```diff
@@ -12,8 +12,8 @@ PACKAGE_ARCH = "${MIDDLEWARE_ARCH}"
 
 SRC_URI = "${CMF_GITHUB_ROOT}/xdialserver;protocol=${CMF_GIT_PROTOCOL};branch=develop"
 
-# Aug 06, 2025
-SRCREV = "3202ca6bbf15d9567f3ab746ba37ce7eccd18c85"
+# Sep 3, 2025 (from develop)
+SRCREV = "206d13783924f8ea90d48dbbcc9c199e45ffa783"
```

Comment:
Why is this from develop and not getting tagged? @apatel859

Reply:
@anand-ky  we have to fix this i realize while reviewing that for xdial repo is not using srvrev from meta-middleware-generic so ask to create a sperate ticket and fix it. You can merge this for now.